src/seedu/addressbook/data/person/Person.java

```diff
@@ -14,8 +14,12 @@
     private Phone phone;
     private Email email;
     private Address address;
-
+    private int sequenceNumber;
+    
     private final UniqueTagList tags;
+    
+    private static int nextSequenceNumber = 0;
```

Comment:
Is this suppose to be a constant? If yes (obv), do adhere the java coding conventions for constants ~